src/gui_common/DraggableScrollContainer.cs

```diff
@@ -30,6 +27,7 @@ public partial class DraggableScrollContainer : ScrollContainer
     private bool centering;
 
 #pragma warning disable CA2213
+    [Export]
     private Control content = null!;
```

Comment:
Hang on, this seems like a behaviour change. It used to be that if this is unset then the first child is used. So shouldn't ready have `if (content == null!) content = GetChild(0)`?